cmd/res/configuration.toml

```diff
@@ -0,0 +1,104 @@
+[Writable]
+LogLevel = "INFO"
+  # Example InsecureSecrets configuration that simulates SecretStore for when EDGEX_SECURITY_SECRET_STORE=false
+  # InsecureSecrets are required for when Redis is used for message bus
+  [Writable.InsecureSecrets]
+    [Writable.InsecureSecrets.DB]
+    path = "redisdb"
+      [Writable.InsecureSecrets.DB.Secrets]
+    [Writable.InsecureSecrets.Camera001]
+    path = "credentials001"
+      [Writable.InsecureSecrets.Camera001.Secrets]
+      username = "administrator"
+      password = "Password1!"
+    # If having more than one camera, uncomment the following config settings
```

Comment:
Either comment out the following block or change the comment

Reply:
Comment out the example config.